language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/util/RegexpCompletionProvider.java

```diff
@@ -481,6 +490,12 @@ private static boolean isReAtom(NonTerminalNode nodeAtCursor, BallerinaCompletio
                 || nodeAtCursor.kind() == SyntaxKind.RE_CAPTURING_GROUP;
     }
 
+    private static boolean isCapturingGroupStart(NonTerminalNode nodeAtCursor, BallerinaCompletionContext ctx) {
+        return nodeAtCursor.kind() == SyntaxKind.RE_CAPTURING_GROUP
```

Comment:
Is this conidtion valid for `re (  <cursor>)` ?

Reply:
No. It should not valid.